roles/datastore/datastore_setup/vars/main.yml

```diff
@@ -14,3 +14,8 @@ user_home_dir: /home/{{ custom_user }}/
 repo_dir: IATI-Datastore/
 
 wsgi_file_name: datastore.wsgi
+
+apache2_conf_file: apache2.conf
+
+# This variable may be refactored away later if it becomes cumbersome as the number of playbooks increases and we no longer wish to specify the version of mod-wsgi to use for each project
```

Comment:
This comment should be located in the `apache2` module rather than the Datastore one.

(if there become a lot of modules using this, duplicating the comment in each module that uses it seems a but non-DRY; and having it just present in `datastore_setup` means that it's unlikely to be seen)